### DISCUSSION TITLE
请问是否有命令仅清理指定项目?

### DISCUSSION THREAD

**Discussion by cjxx2016** [xmake-io/xmake#2611] [Q&A] [answered]
目前我使用 xmake clean 命令可以清理工程下的所有项目, 我希望仅清理特定项目, 请问这可以做到么? 比如有个EXE项目依赖了一个lib 而我仅需要清理并重编EXE而保留lib 的编译结果. 这样可以节省一些.编译时间. 感谢.

**Comment by waruqi** [comment, marked as answer]
xmake clean targetname

指定 target 清理，dev/master 版本会仅仅清理当前target，不清理依赖。见：https://github.com/xmake-io/xmake/issues/2552

**Reply by cjxx2016** [reply]
收到, 非常感谢.
